authorization-dotnet/AuthorizationPolicyProviders/AuthorizationPolicyProviders/Controllers/HomeController.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using AuthorizationPolicyProviders.Authorization.Attributes;
+
+namespace AuthorizationPolicyProviders.Controllers;
+
+public class HomeController : Controller
+{
+    public IActionResult Index()
+    {
+        return View();
+    }
+
+    [LoyaltyAuthorize("PriorityCheckIn")]
+    public IActionResult PriorityCheckIn()
+    {
+        return View();
+    }
+}

```

Comment:
@saraelsa 
Can we add another method here that shows the use of `[Authorize("Loyalty:PriorityCheckIn")]`